helpers/core.js

```diff
@@ -114,15 +114,13 @@ var helpers = {
 		var expr = resolved,
 			result;
 
-		if ( !! expr && utils.isArrayLike(expr)) {
-			result = utils.getItemsFragContent(expr, options, options.scope, asVariable);
-			return options.stringOnly ? result.join('') : result;
-		}
-		else if(isIterable(expr)) {
+		if(isIterable(expr)) {
 			result = [];
 			each(expr, function(value, key){
 				aliases = {
-					"%key": key
+					"%key": key,
```

Comment:
I don't think this needs to handle objects, since there is already a branch below for `expr instanceof Object`. Maybe the check here should just be `Array.isArray(expr)`.

Reply:
Very good point. Can you see any potential problems with just getting rid of the `expr instanceof Object` branch and letting this handle both? Seems like there is a lot duplicated logic for no reason.

Reply:
The `isIterable(expr)` case doesn't cover plain objects.